plugins/footnote/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "author": "Discourse",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint-config-discourse": "^3.4.0"
+  }
+}
```

Comment:
This file is probably not needed either now that footnote is in core, it should follow core's eslint config.